packages/ipfs-daemon/src/ipfs-daemon.ts

```diff
@@ -91,17 +90,8 @@ export class IpfsDaemon {
           allowNew: configuration.ipfsCreateIfMissing,
         },
         repo: configuration.ipfsPath,
-        ipld: { codecs: [dagJose] },
         libp2p: {
-          config: {
-            dht: {
-              enabled: false,
-              clientMode: !configuration.ipfsDhtServerMode,
-            },
-            pubsub: {
-              enabled: configuration.ipfsEnablePubsub,
```

Comment:
Does this have some impact?

Reply:
I believe this config is only for js-ipfs if I'm not mistaken, and we disabled the js-ipfs tests

Reply:
Good catch. I do not know, to be honest. Let me get it back indeed. I finally figured the types.